artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionBridge.java

```diff
@@ -278,7 +279,12 @@ private void setupNotificationConsumer() throws Exception {
                                                    createPermissiveManagementNotificationToFilter() +
                                                    ")");
 
-         sessionConsumer.createQueue(new QueueConfiguration(notifQueueName).setAddress(managementNotificationAddress).setFilterString(filter).setDurable(false).setTemporary(true));
+         sessionConsumer.createQueue(new QueueConfiguration(notifQueueName)
+                                        .setAddress(managementNotificationAddress)
+                                        .setFilterString(filter)
+                                        .setDurable(false)
+                                        .setTemporary(true)
+                                        .setRoutingType(RoutingType.MULTICAST));
```

Comment:
was this an actual issue? or you are just making sure?

the address being a Multicast, was it creating ANYCAST queues in some cases?

Reply:
Semi related and making sure. I saw this issue presenting very consistently on a broker cluster in a TEST environment. Turn out they had configured default-routing-type = ANYCAST as an address setting and a match on "#" so that the notifications got load balanced between brokers, sort of mimicking the issue I'm going after with this PR.

I figured there ought to be no use case for setting up cluster notifications with ANYCAST semantics so I added this.